client/state/analytics/actions/with-analytics.js

```diff
@@ -1,11 +1,6 @@
-/**
- * External dependencies
- */
-import { curry, get, merge } from 'lodash';
-
 const mergedMetaData = ( a, b ) => [
-	...get( a, 'meta.analytics', [] ),
-	...get( b, 'meta.analytics', [] ),
+	...( a?.meta?.analytics ?? [] ),
+	...( b?.meta?.analytics ?? [] ),
```

Comment:
The `a` and `b` are always there, being actions. We need only one optional chaining: `a.meta?.analytics`

Reply:
Good call 👍  Updated in https://github.com/Automattic/wp-calypso/pull/51895/commits/999795392ca1de55c05b629fee056d0539a2b595